index.html

```diff
@@ -229,38 +229,71 @@ <h3>
             <dt id="data-integrity-1" class="spec">Verifiable Credential Data Integrity 1.0</dt>
             <dd>
               <p>
-This specification defines how to express proofs of integrity, such as digital
-signatures or proofs of existence, for bounded documents, such as verifiable
-credentials. Concrete serializations will be provided based on <a
-href="https://w3c.github.io/vc-data-model/#json-web-token">VC-JWT</a>, the <a
-href="https://w3c-ccg.github.io/lds-ed25519-2020/">Ed25519 Cryptosuite</a>, 
-the <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">Secp256k1
-Cryptosuite</a>, and for <a href="https://github.com/w3c-ccg/lds-jws2020/">
-JSON Web Signature 2020</a>. Concrete serializations based on the <a
-href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Cryptosuite</a> might be provided, if an <a
-href="https://identity.foundation/bbs-signature/">IETF BBS+ RFC</a> is published
-in time. Concrete serializations for VC-JWP might be provided, if an <a
-href="https://json-web-proofs.github.io/json-web-proofs/">IETF JWP RFC</a> is
-published in time. 
-The final output specifications might not map directly to
-the list of input drafts. Other output serializations covering cryptographic
-primitives published in IETF RFCs not mentioned in this paragraph might be
-produced.
+This family of specifications defines how to express proofs of integrity for
+verifiable credentials using a number of concrete serializations for each of the
+defined syntaxes. The specific set of concrete serializations included will be
+determined by the Working Group. The following are a non-exhaustive selection
+of expected input documents:
               </p>
-
-              <p class="draft-status"><b>Draft state:</b>
-                <a href="https://w3c-ccg.github.io/data-integrity-spec/">Data Integrity</a>,
+              <p class="input-documents"><b>Container Formats:</b>
                 <a href="https://w3c.github.io/vc-data-model/#json-web-token">VC-JWT</a>,
-                <a href="https://w3c-ccg.github.io/lds-ed25519-2020/">Ed25519 Cryptosuite</a>,
-                <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">Secp256k1 Cryptosuite</a>,
-                <a href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Cryptosuite</a>,
-                <a href="https://github.com/w3c-ccg/lds-jws2020/">JSON Web Signature 2020</a>              
+                <a href="https://w3c-ccg.github.io/data-integrity-spec/">Data Integrity</a>
+              </p>
+              <p class="input-documents"><b>Cryptosuites:</b>
+                <a href="https://github.com/w3c-ccg/lds-jws2020/">JSON Web Signature</a>,
+                <a href="https://w3c-ccg.github.io/lds-ed25519-2020/">EdDSA</a>,
+                <a href="https://w3c-ccg.github.io/di-ecdsa-secpr1-2019/">NIST ECDSA</a>,
+                <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">Koblitz ECDSA</a>
```

Comment:
related to my comment above on input document being published/stable standards, I suggest to move all community drafts from this section to the section below on "produce W3C Recommendations for the following documents"

Reply:
reflecting a bit more, I think if we were to add another W3C recommendation in this group, we would need to recharter, and I don't think we want to open another pandora box of producing unpredictable number of additional recommendations. So we would want to state explicitly that this WG will take over from CCG and work on data integrity 1.0 if it will.

Reply:
I would suggesting moving this comment to an issue, and accepting this PR as is.

Reply:
Note the comment about what an "input document" at W3C means: https://github.com/w3c/vc-wg-charter/pull/77/files#r812415276

Reply:
Input document can be a standard that is published by another standards body that has been accepted by the W3C.

Reply:
IMO, we should avoid trying to do too much in 1 PR... it makes the comments and intention of the PR unreadable. This PR is about matching the web app sec wg style, suggested changes input items should be moved to issues, so we don't forget to address them when merging a PR that does restructuring.

Reply:
ok, I opened an issue #88 to discuss input document content.